packages/app-config-writer/src/variants-config/package-json.ts

```diff
@@ -43,13 +40,26 @@ export async function addVariantsManagementScript(
         urlParameters['sap-client'] = sapClient;
     }
 
-    const url = await getRTAUrl(basePath, getUI5UrlParameters(urlParameters), yamlPath);
+    const url = await getRTAUrl(basePath, getUI5UrlParameters(urlParameters), ui5YamlFileName);
+    const serveCommand = await getRTAServe(basePath, ui5YamlFileName, fs);
 
     if (!url) {
         throw new Error(`${ERROR_MSG} No RTA editor specified in ui5.yaml.`);
     }
 
-    packageJson.scripts['start-variants-management'] = `fiori run --open "${url}"`;
+    const startVariantsManagementScriptOld = packageJson.scripts['start-variants-management'] ?? undefined;
+    const startVariantsManagementScriptNew = `${serveCommand} --config ./${basename(ui5YamlFileName)} --open "${url}"`;
```

Comment:
@tobiasqueck @CG67 do we want to have --config mentioned in the script at all times or should we omit the default `ui5.yaml`).I'm indifferent. Opinions?

Reply:
I'd omit `--config` if it is using `ui5.yaml`.

Reply:
Done in 2dadf466926177a6fbf6ad132909f6c84ba17a1d